app/src/main/java/com/github/se/gatherspot/model/EventsViewModel.kt

```diff
@@ -36,6 +40,14 @@ class EventsViewModel : ViewModel() {
     registeredTo = eventFirebaseConnection.fetchRegisteredTo()
   }
 
+  suspend fun fetchEventsFromFollowedUsers() {
+    val ids =
+        FollowList.following(
+            FirebaseAuth.getInstance().currentUser?.uid ?: "CpsyL2BH9TTQKEfpDC3YwZB6NLE2")
```

Comment:
Risky to put a magic value over here. It stills works but it wouldn't change if the testLogin changes.

Reply:
True, I'll modify this